docs/api/model.rst

```diff
@@ -15,10 +15,13 @@ Language Modeling
     awd_lstm_lm_1150
     awd_lstm_lm_600
     AWDRNN
+    train.AWDRNN
```

Comment:
these should be listed in a separate page. see example for gluonnlp.data.batchify.